saplings/circuits/src/index.js

```diff
@@ -16,11 +16,15 @@
 
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { registerApp } from 'splinter-saplingjs';
+import { registerApp, registerConfigSapling } from 'splinter-saplingjs';
 
 import './index.css';
 import App from './App';
 
-registerApp(domNode => {
-  ReactDOM.render(<App />, domNode);
+registerConfigSapling('circuits', () => {
+  if (window.location.pathname === '/circuits') {
+    registerApp(domNode => {
+      ReactDOM.render(<App />, domNode);
+    });
+  }
```

Comment:
I'm not 100% sure this is the correct solution. canopyJS make a distinction between `configSapling` and `userSapling`. And circuit sapling is a `userSapling`, so I think it may cause problems/be incorrect to use `registerConfigSapling` here. @dplumb94 @dnewh, what do you think?

Reply:
I see. I was using profile as a reference since that was the only one that rendered properly. Guess you really only need to check the path name. Changed to
```
registerApp(domNode => {
  if (window.location.pathname === '/circuits') {
    ...
  }
});
```